js/components/skipControls.js

```diff
@@ -146,6 +156,13 @@ class SkipControls extends React.Component {
       buttonStyle.pointerEvents = 'none';
     }
 
+    var playButtonDetails = Utils.getPlayButtonDetails(this.props.controller.state.playerState);
+    var playIcon = playButtonDetails.icon;
```

Comment:
Another subjective comment, but I think this might be cleaner if we used `playButtonDetails.icon` directly in the component in order to avoid the extra lines. Otherwise I think these should be declared as `const`

Reply:
Will add